packages/main/src/Table.ts

```diff
@@ -463,6 +483,88 @@ class Table extends UI5Element {
 		this.fireEvent<TableRowClickEventDetail>("row-click", { row });
 	}
 
+	_ondragenter(e: DragEvent) {
```

Comment:
_ondragenter, _ondragover, _ondragleave are pretty much the same implementation as in List.ts.

So maybe there is a way to reuse the code in a Utility or something?